cmd/img_mgmt/include/img_mgmt/img_mgmt.h

```diff
@@ -37,6 +37,13 @@ extern "C" {
 #define IMG_MGMT_ID_CORELOAD        4
 #define IMG_MGMT_ID_ERASE           5
 
+/*
+ * IMG_MGMT statuses.
+ */
+#define IMG_MGMT_STATUS_UPLOAD_START            0x80
```

Comment:
Why starts from 0x80? I can expect that this is related to MGMT_STATUS_CMD_xxx which will be added to the result status.

Reply:
Exactly, this is due to MGTM_STATUS* codes that are 0x01 and 0x02.